Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure/token.go

```diff
@@ -1,12 +1,19 @@
 package azure
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"encoding/base64"
+	"fmt"
 	"net/http"
 	"net/url"
 	"strconv"
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
I've added a new commit; it vendors sub-dependencies (`pkcs12`, `jwt-go`, and `check.v1`), includes the sub-dependencies' test code (keeping in line with how autorest was originally vendored), and rewrites paths.

Hopefully this resolves any concerns.

LMK and I'll squash it.